### DISCUSSION TITLE
Is there any file to show Termux's version?

### DISCUSSION THREAD

**Discussion by dontknowhy** [termux/termux-app#2914] [Q&A]
Is there any file allow program to know Termux's version?
I what to know the Termux's version without $TERMUX_VERSION.
And i need to know the Termux's version in a container.(mounted /data)
(do not ask me for why)

**Comment by agnostic-apollo** [comment]
Read `$TERMUX_VERSION` from `/data/data/com.termux/files/usr/etc/termux/termux.env` if using github builds, not released on F-Droid yet.

https://github.com/termux/termux-app/commit/03e1d14e1e65e054eeeb0ad672a46c2c17254d3e

>do not ask me for why

Why not? Only you can ask questions, huh!

**Comment by RalfWerner** [comment]
The termux-version is also in the file: `b=base.apk`, which exists after installation in the path: `p=$nl/../../` and can be read with:
`aapt d badging $apk` (`apk=$p$b`and`nl=~/../../lib`). In last GitHub version you could use `apk=$TERMUX_APP__APK_PATH`.

**Comment by dontknowhy** [comment]
> The termux-version is also in the file: `b=base.apk`, which exists after installation in the path: `p=$nl/../../` and can be read with:
> `aapt d badging $apk` (`apk=$p$b`and`nl=~/../../lib`). In last GitHub version you could use `apk=$TERMUX_APP__APK_PATH`.

But i'm always using F-Droid version…

**Reply by RalfWerner** [reply]
`$TERMUX_APP__APK_PATH` will be included in the next F-Droid release. For most devices there is a _workarounds_ for `$nl`, if this does not already exist on your device.
